default.nix

```diff
@@ -0,0 +1,9 @@
+{pkgs ? import <nixpkgs> {}}: let
+  lib = pkgs.lib;
+  matchbox = pkgs.buildGoModule {
+  name = "matchbox";
+  src = lib.cleanSource ../matchbox;
+  vendorHash = "sha256-sVC4xeQIcqAbKU4MOAtNicHcioYjdsleQwKWLstnjfk";
```

Comment:
How are you imaginging this being updated? I don't expect to maintain or bump a hash. If the aim is for this to be an alternative to using the Makefile during local iteration/development, vendorHash seems counter to quick iteration

Reply:
The vendor hash must be set per release tag because they are releases.
You can also set `vendorHash` to `null` in the main branch but `nix-build` won’t work out of the box because `go mod vendor` must be run first.

Reply:
For an in-project development Nix config, I think the config should just build from the local source. As in, if you edit some Go files or dependencies locally, running nix-build should just build it without someone having to fiddle with hashes. I'd prefer real releases and vendor pinning be done in upstream nixpkgs.

Do you see a usage story that aligns with this or no?

Reply:
Sounds fair.